posts/serializers.py

```diff
@@ -42,3 +42,23 @@ class Meta:
             'unlisted'
         ]
 
+class CommentSerializer(serializers.ModelSerializer):
+    # type is only provided to satisfy API format
+    type = serializers.CharField(default="comment", source="get_api_type", read_only=True)
+    # public id should be the full url
+    id = serializers.CharField(source="get_public_id", read_only=True)
```

Comment:
Perhaps this is another question to ask, but is the ID(url) available for write? I don't think it's specified as of whether it's read-only or not. I would hope it's read-only.